<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FieldItem</class>
 <widget class="QWidget" name="FieldItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>240</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#FieldItem {
    background-color: white;
    border: 1px solid #cccccc;
}

QLabel {
    color: black;
    background: none;
    border-bottom: 1px solid #cccccc;
}

QPushButton {
    background-color: #4CAF50;
    color: white;
    border: none;
    border-radius: 0;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="image_label">
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>130</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>280</width>
       <height>130</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #f5f5f5;</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="name_label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 5px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="details_label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 5px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="view_button">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>View More</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
